reference/shipping.v3.yml

```diff
@@ -355,11 +407,17 @@ components:
         - commodity_description
         - international_shipping
         - hs_codes
+    shippingSettings:
+      title: shippingSettingsRequest
+      description: Data about the shipping settings object.
+      type: object
+      properties:
+        checkout:
+          $ref: '#/components/schemas/countryListStrategy'
```

Comment:
Looks like one final thing to resolve here, which relates to avoiding mixing `countryListStrategy` with `checkout` for our `$ref` value. As I mentioned elsewhere, I think the ultimate solution is this...

```suggestion
    shippingSettings:
      type: object
      properties:
        checkout:
          type: object
          properties:
            country_list_strategy:
              description: The approach for displaying the list of countries at checkout.
              type: string
              enum:
                - DISPLAY_ALL_COUNTRIES
                - DISPLAY_ONLY_SHIPPABLE_COUNTRIES
              example: DISPLAY_ALL_COUNTRIES
```

Worth noting is that I've removed title and description from `shippingSettings` also, since they don't add any meaningful value (and it matches our [Tax_Settings](https://github.com/bigcommerce/docs/blob/d08d3a38e0eb2eda3eb08a5fd049568f3f2d871a/reference/tax_settings.v3.yml#L54) approach).

In addition to this change, we then remove the `countryListStrategy` definition (which is actually a definition for the _checkout_ object) on line 458 since it's now unused.

Another benefit here is that all the information about `shippingSettings` is in the one schema definition rather than jumbled up among the YML definitions.

Reply:
Oh I see - Thanks mate, I've updated this as per your suggestion. I now have a better idea how to do docs tasks😅🙏🍻

Reply:
Thank you sir 🚀